wordpress-mobile/gutenberg-mobile
# Fix for title that is not saving

This fixes #620 

GB side companion PR: https://github.com/WordPress/gutenberg/pull/13967

To test:
- Update your wp-android or wp-ios to point to this branch and make sure to build from source
- Edit or create a new post
- Make sure the title is correctly posted on the site
